services/frontend/pkg/config/defaults/defaultconfig.go

```diff
@@ -72,7 +72,7 @@ func DefaultConfig() *config.Config {
 					"X-HTTP-Method-Override",
 					"Cache-Control",
 				},
-				AllowCredentials: true,
+				AllowCredentials: false,
```

Comment:
`curl "https://localhost:9200/ocs/v1.php/config" -H"Origin: https://aphno.badal" -vk`

Previously:
```
< Access-Control-Allow-Origin: https://aphno.badal
< Access-Control-Expose-Headers: Location
< Access-Control-Expose-Credentials: true
```

With the PR:

```
< Access-Control-Allow-Origin: https://aphno.badal
< Access-Control-Expose-Headers: Location
```
There is no `Access-Control-Expose-Credentials` header. Since it is changed to false, is it expected to not show in header as `Access-Control-Expose-Credentials: false`?

CC @dragonchaser